backend/app/adapter/db/userurl_test.go

```diff
@@ -11,16 +9,46 @@ import (
 	"github.com/byliuyang/app/mdtest"
 )
 
-func TestNewUserURLRelationSQL(t *testing.T) {
+func TestListURLSql_FindAliasesByUser(t *testing.T) {
+	var mockedEmptyAliases []string
+	var mockedAliases []string
+	mockedUser := entity.User{
+		Name:           "mockedUser",
+		Email:          "test@example.com",
+		LastSignedInAt: mustParseSQLTime("2019-05-01 08:02:16"),
+		CreatedAt:      mustParseSQLTime("2019-05-01 08:00:16"),
+		UpdatedAt:      mustParseSQLTime("2019-05-01 08:02:16"),
+	}
+
+	alias := "abcd-123-xyz"
+	mockedAliases = append(mockedAliases, alias)
+
 	testCases := []struct {
-		name      string
-		user      entity.User
-		alias     string
-		expHasErr bool
+		name             string
+		tableRows        *mdtest.TableRows
+		user             entity.User
+		hasErr           bool
+		expectedResponse []string
```

Comment:
```suggestion
		expectedAliases []string
```
This renaming is to improve the clarity of the code.